docs/cross_toml.md

```diff
@@ -29,5 +29,7 @@ volumes = ["VOL1_ARG", "VOL2_ARG"]
 passthrough = ["VAR1", "VAR2"]
 xargo = false
 image = "test-image"
+dockerfile = "./Dockerfile"
+context = "."
```

Comment:
This should maybe be more explicit, or even added as its own table together with dockerfile

So

`dockerfile = {file = "./Dockerfile", context = "." }`

Could even make it take either a table or string